### PR TITLE
changed the caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @1tejaswini1
- [ ] @GVEERABABU
- [ ] @Radhikamuvva
- [ ] @aharshit93
- [ ] @akkp-cts
- [ ] @bsuppala
- [ ] @ericalloyd
- [ ] @mvikas99
- [ ] @prasobhHERE
- [ ] @rasrinivasa
- [ ] @rkumarreddy
- [ ] @shahim22
- [ ] @stevenvelasquez
- [ ] @vbasvabovina
- [ ] @vikas-khindkar
- [ ] @auharrison-clgx

### PR DESCRIPTION
**Pull request by vbasvabovina-clgx**
Fixes #2

### CODE REVIEW

File: _slides/13-vbasvabovina.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Woof woof!
```
Comment (auharrison-clgx) [inline _slides/13-vbasvabovina.md:6]: Looks good.

Comment (auharrison-clgx) [reply _slides/13-vbasvabovina.md:6]:
```suggestion
arf arf!
```

Comment (vbasvabovina-clgx) [inline _slides/13-vbasvabovina.md:6]: Thank you

**Review by auharrison-clgx — APPROVED** [review]
💯

**Pull request merged**